src/index.js

```diff
@@ -16,6 +16,7 @@ yargs
       }),
     args => mainProcess(args)
   )
+  .boolean('yes')
```

Comment:
The disadvantages of the boolean option is that it is not print as an available option when you do `readme --help`.
It would be great to have an alias for this option, so we can do `readme --yes` or `readme -y`